scripts/convert_gen2_to_gen3_hsc.sh

```diff
@@ -8,7 +8,30 @@
 
 # Run the gen2 to gen3 conversion code
 # This conversion is done in-place so the same path is used for the gen2 and gen3 repos.
-butler convert hsc -i lsst.obs.subaru.HyperSuprimeCam --gen2root ./hsc
+
+# Clean out old gen3 repo if it is there
+if [ -f "hsc/butler.yaml" ]; then
```

Comment:
What files in the `hsc/` directory are you trying to retain?

Reply:
All the gen2 files.  I'm making the gen3 in place.  Maybe it would be simpler to make a hsc_gen3 directory instead?